### PR TITLE
General maintenance

### PR DESCRIPTION
**Pull request by gigatexel**
Done:

- Add support for hassfest & Validate-HACS
- Made necessary changes to comply to hassfest and Validate-HACS checks
- Corrected some keys in manifest file
- Fixed numerous typo's

### CODE REVIEW

File: custom_components/loxone/manifest.json
Comment (JoDehli) [inline custom_components/loxone/manifest.json:1]: Why do you us remove the render_readme entry?

File: hacs.json
Comment (JoDehli) [inline hacs.json:1]: Why do you remove the domains? Is it not needed anymore?

File: custom_components/loxone/manifest.json
Comment (gigatexel) [reply custom_components/loxone/manifest.json:1]: It is not a valid entry for manifest.json (see https://developers.home-assistant.io/docs/creating_integration_manifest/)

It should be included in hacs.json (see https://hacs.xyz/docs/publish/start/). It's also already there :-)

File: hacs.json
Comment (gigatexel) [reply hacs.json:1]: domains is not a valid parameter for hacs.json (see https://hacs.xyz/docs/publish/start/). I also could not find a reference/docs for this.

**Review by JoDehli — APPROVED** [review]
Thanks for your help

**Comment by JoDehli** [comment]
@gigatexel at the moment I work on the improvement of lights. Until this is finished we should extend pyloxone. This little improvement should be made for every platform and then I want to complete rewrite the websocket connection.

**Comment by JoDehli** [comment]
@gigatexel I you have time and want to help you could help me with the light improvements https://github.com/JoDehli/PyLoxone/tree/feature/light-seperation

**Comment by gigatexel** [comment]
> @gigatexel I you have time and want to help you could help me with the light improvements https://github.com/JoDehli/PyLoxone/tree/feature/light-seperation

I've got time and keen to learn :-)
What's the idea?

**Comment by gigatexel** [comment]
Maybe activate the Discussions section?

**Pull request merged**